example-apps/python-flask-registration-forms.json

```diff
@@ -0,0 +1,211 @@
+{
+  "variables": {
+    "defaultTenantId": "d7d09513-a3f5-401c-9685-34ab6c552453",
+    "adminEmail": "admin@example.com",
+    "adminPassword": "password",
+    "flaskAppId": "#{UUID()}",
+    "registrationFormId": "#{UUID()}",
+    "geoFieldId": "#{UUID()}",
+    "maxPriceFieldId": "#{UUID()}",
+    "minPriceFieldId": "#{UUID()}"
+  },
+  "licenseId": "nDB+bOBNk0S6Bt9yicu33PdBms3phj013xEMvDwvFHw",
```

Comment:
No license Id should be in here. Or you can just say something like "TODO enter your license".

You should probably rotate this license Id too, as who knows if this got picked up. (So delete it and create a new one in account.fusionauth.io.)

Reply:
thanks, I had meant to do that before committing